src/test/java/cloud/fogbow/ras/core/OrderStateTransitionerTest.java

```diff
@@ -70,6 +69,48 @@ public void testTransitionToChangeOrderStateOpenToSpawning() throws UnexpectedEx
         Assert.assertNull(openOrdersList.getNext());
     }
 
+    // test case: When calling the transition() method, it must change the state of Order
+    // from the remote order passed as parameter to a specific OrderState. This Order
+    // defined originally with Open, will be changed to the Spawning state, removed
+    // from the open orders list, and added to the spawning orders list.
+    @Test
+    public void testTransitionToChangeOrderStateOpenToSpawningWithRemoteProvider() throws Exception {
```

Comment:
we have discussed that you should only check if transition makes the expected calls and create another test to really test the doTransition method, right?

Reply:
Right. Changed that.